src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -96,6 +127,26 @@ public static IDataTransform Create(IHostEnvironment env, IDataView input, strin
             return new TensorFlowTransform(env, TensorFlowUtils.GetSession(env, model), source, names, TensorFlowUtils.IsSavedModel(env, model) ? model : null, false).MakeDataTransform(input);
         }
 
+        /// <summary>
+        /// Utility method to create <see cref="TensorFlowTransform"/> with retraining capabilities.
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="input">Input <see cref="IDataView"/>. This is the output from previous transform or loader.</param>
+        /// <param name="model">Path to the TensorFlow model. </param>
+        /// <param name="names">Name of the output column(s). Keep it same as in the Tensorflow model.</param>
+        /// <param name="source">Name of the input column(s). Keep it same as in the Tensorflow model.</param>
+        /// <param name="args">Argument object containing training related parameters.</param>
+        /// <returns></returns>
+        public static IDataTransform Create(IHostEnvironment env, IDataView input, string model, string[] names, string[] source, TrainingArguments args)
```

Comment:
>IDataTransform [](start = 22, length = 14)

I think we trying to move away from IDataTransform, and only reason why it still exist is because we not fully switched away from it. Can you make it IDataView?

Reply:
I will have to change all `create` method that are producing `IDataTransform` and `LodableClass` calls above, right?

---
In reply to: [221764913](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221764913) [](ancestors = 221764913)

Reply:
```LodableClass``` points on already existing method. You either need to add another attribute, which would point on this one, or you can return here whatever you want, since it's not tied with our DI.

---
In reply to: [221768980](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221768980) [](ancestors = 221768980,221764913)

Reply:
I actually removed this constructor after merging `TrainingArguments` with `Arguments` class.

---
In reply to: [221770021](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221770021) [](ancestors = 221770021,221768980,221764913)